python/PiFinder/camera_interface.py

```diff
@@ -162,5 +162,12 @@ def get_image_loop(
                         filename = f"{utils.data_dir}/captures/{filename}.png"
                         self.capture_file(filename)
                         console_queue.put("CAM: Saved Image")
+                    if command.startswith("stop"):
```

Comment:
I'm not sure, but I expect that stopping the camera will generate an error as the get_image_loop will continue to run an try to call the `capture` method.  This is crashing the loop and preventing new images from coming through even after restarting?

* Maybe a state variable on CameraInterface class to keep track of the state of the camera, which can be checked before calling the capture method in the get_image_loop
* Or, a try/except which catches whatever the specific exception is when you try to call    `camera.capture_request()` in the camera_pi.py file on line 84